### DISCUSSION TITLE
Daily 19/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#178] [General]
Ayer modifiqué los modelos para dejarlo en una clase por tabla de base de datos, después de ver los videos otra vez me di cuenta que si todos van a utilizar el mismo método de corrección lo mejor es ir por lo seguro. También estuve probando FastAPI y pude hacer que haya conexión entre el back-end y el front-end que no piden pero aprovecho para ver qué tan difícil sería implementarlo. También armé algunas issues sobre cosas que hay que hacer. Hoy pienso ver si se me ocurren más issues ya con queries que haya que armar para que cada uno tome al menos uno para completar.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve estudiando para el examen de GP.
Hoy continuaré con el estudio de GP y comenzaré con JS de Full Stack.

**Comment by Brian-Almada** [comment]
Ayer estudie un poquito de ingles, vi algo de WP y me puse a revisar algunas ideas para cuando implementemos JS en el proyecto (esto mismo quedó asentado en discussions). Ahora estoy haciendo unas actividades de ingles y luego haré unos ejercicios de python.

**Comment by AhumadaCeleste** [comment]
Ayer estudie para el examen de proyecto lo cual finalice hoy para poder hacer repasos cortos diarios y legar bien. Hoy estuve en clase de ingles, complete la lección de vos pasiva y activa. ahora voy a continuar un poco con wordpres. Mañana quisiera terminar de afianzar boobstarps para iniciar con java. Comencé a ver algunos videos antes de dormir como introducción en el tema y poder comenzar a crear issues.
